engine/flink/kafka-util/src/main/scala/pl/touk/nussknacker/engine/kafka/KafkaSinkFactory.scala

```diff
@@ -29,8 +29,9 @@ class KafkaSinkFactory(config: KafkaConfig,
              @NotBlank
              topic: String
             )(metaData: MetaData): Sink = {
-    val serializationSchema = schemaFactory.create(topic, config)
-    new KafkaSink(topic, serializationSchema, s"${metaData.id}-$topic")
+    val _topic = NamespaceUtil.alterTopicNameIfNamespaced(topic, config)
```

Comment:
Maybe `topicName` instead of `_topic`?

Reply:
Or better `namespacedTopic`? What do you think?

Reply:
But when topic is wihout namespace then it's not namespaced ;)

Reply:
yes, but `topicName` suggests that in this line we get the property `name` of the object `topic`, while both of them are simply strings - if we want to be so meticulous with naming